cli/args/mod.rs

```diff
@@ -1092,7 +1122,19 @@ impl CliOptions {
             value,
           })
         }
-        None => None,
+        None => {
+          if let Some((path, import_map)) =
+            self.maybe_external_import_map.as_ref()
+          {
+            let path_url = deno_path_util::url_from_file_path(path)?;
```

Comment:
Will this condition ever be hit? I feel like `resolve_specified_import_map_specifier` would always be `Some` in this case?

Reply:
I thought so too, but that function only uses `--import-map` flag (it warns if you have an `importMap` field in deno.json, but it doesn't actually do anything with the value)

https://github.com/denoland/deno/blob/32b57f7b82d26c40586fad1d08d5bd4f3061e6a8/cli/args/mod.rs#L1872-L1892

Just to make sure, I added some logs locally and confirmed it does end up hitting this branch